src/index.js

```diff
@@ -1,6 +1,32 @@
-/**
-* Put your JavaScript here
-*/
+let state = {
+  todos: [
+    { title: 'First todo', complete: false },
+    { title: 'Second todo', complete: true },
+    { title: 'Third todo', complete: true },
+  ],
+};
+
+document.onload = init();
```

Comment:
Indeed! I wouldn't say it's not the right thing to do, but we should be aware of the consequence. On this site, it makes no difference; on bigger apps, we spend a lot of time on performance perception and that could make a big difference.